setup/scripts/health-check.sh

```diff
@@ -1,14 +1,34 @@
 #!/bin/bash
 
-echo -e "${BLUE}Performing a health check...${NC}\n"
+check_services() {
```

Comment:
Could also look at using `docker-compose -d --wait` in the future - https://maciejwalkowiak.com/blog/docker-compose-waiting-containers-ready/

Reply:
Nice, wasn't familiar.
I tried it now locally but it doesn't work. Probably b/c services are stuck on "starting" and not "healthy".
Creating a follow-up ticket